nodewatcher/modules/frontend/ip_space/templates/network/ip_space/ip_space.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% contextblock %}
+    {% load i18n sekizai_tags html_tags url_tags partial_tags leaflet_tags %}
+
+    {% add_data "css_data" "css/ipspace.css" %}
+    {% add_data "js_data" "js/d3.v4.js" %}
+    {% add_data "js_data" "js/point.js" %}
+    {% add_data "js_data" "js/hilbert2d.js" %}
+    {% add_data "js_data_end" "js/jquery.tipsy.js" %}
+    {% add_data "js_data_end" "js/draw_cidrs.js" %}
+    {% add_data "js_data_end" "js/main.js" %}
+
+    {% trans "Ip space" as title %}
+{% endcontextblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-2">
+            <div class="dropdown">
+                <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{% trans "Zoom to subnet" %}<span class="caret"></span></button>
+                <ul class="dropdown-menu" id="topnodes">
+                    <li id="cidr"><a href="#" value="10.0.0.0/8">{% trans "Loading data" %}</option>
```

Comment:
Don't use `#` in links, because they can destroy potential anchor information.

Why is this a link anyway?

Reply:
Yes it was some weird html(closed </option> even tough there is no start), i fixed it now, it need to be a to be drawn correctly in the dropdown however.